Lib/ctypes/util.py

```diff
@@ -324,6 +322,20 @@ def test():
             print(cdll.LoadLibrary("libcrypto.dylib"))
             print(cdll.LoadLibrary("libSystem.dylib"))
             print(cdll.LoadLibrary("System.framework/System"))
+        # issue-26439 - fix broken test call for AIX
+        elif sys.platform.startswith("aix"):
+            from ctypes import CDLL
+            RTLD_MEMBER =  0x00040000
```

Comment:
Please use os.RTDL_MEMBER instead of an hardcoded constant.

Reply:
done.